web/docs/guides/websockets.md

```diff
@@ -0,0 +1,129 @@
+---
+title: WebSockets
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# WebSocket support
+Wasp provides a fully integrated WebSocket experience by utilizing [Socket.IO](https://socket.io/) on the client and server. We handle making sure your URLs are correctly setup, CORS is enabled, and provide a useful `useSocket` abstraction for use in React components.
+
```

Comment:
Maybe we could mention here, as a NOTE, that if you want, you can still roll your own sockets, via `serverSetupFn`?

Reply:
The most advanced users that might want that will probably figure out that they can do it anyways? It might confuse the not so advanced users.

Reply:
Yes, or if they have some special needs. But you decide, if you think it is redundant and clutters the docs here, let's skip, it is ok with me.